pkg/client/cmd/root.go

```diff
@@ -46,9 +46,9 @@ func NewKamelCommand(ctx context.Context) (*cobra.Command, error) {
 		Context: ctx,
 	}
 	var cmd = cobra.Command{
-		Use:                    "kamel",
-		Short:                  "Kamel is a awesome client tool for running Apache Camel integrations natively on Kubernetes",
-		Long:                   kamelCommandLongDescription,
+		Use:   "kamel",
+		Short: "Kamel is a awesome client tool for running Apache Camel integrations natively on Kubernetes",
+		Long:  kamelCommandLongDescription,
```

Comment:
This will be reverted by gofmt, best if you set it up on your ide

Reply:
@lburgazzoli it's very strange, but I got the same result when I tried to format manually with gofmt

Reply:
strange, when I do save in vscode, it gets automatically indented. In any case it is not an issue